grid/src/js/controllers/spinner.js

```diff
@@ -0,0 +1,11 @@
+import eventEmitter, {SPINNER} from "../ee";
+export default class spinnerController {
+  constructor(view) {
+    this.view = view;
+
+    eventEmitter.on(SPINNER, (isToogleBackdrop) => {
+      if (isToogleBackdrop === undefined) isToogleBackdrop = true;
```

Comment:
Can we use 
eventEmitter.on(SPINNER, (isToogleBackdrop **= true**) => {
instead?